docs/cloud-protocol/state.reported.aws.json

```diff
@@ -51,6 +49,8 @@
   },
   "roam": {
     "v": {
+      "band": 3,
+      "nw": "NB-IoT GPS",
```

Comment:
```suggestion
      "nw": "NB-IoT",
```

Reply:
The API that we use to retrieve the network mode only returns the current LTE mode. So the GPS string will never be present after https://github.com/nrfconnect/sdk-nrf/pull/6148.